src/predict/greater_kampala_metropolitan_area_kiidp_2012/Greater_Kampala_Metropolitan_Area_KIIDP_2012.shp.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Just curious, could we explore the storage of these kinds of files in GCS or any other appropriate tool?
Same case for the binary files I have seen above in the PR......